host/libvirt_lxc_backend.go

```diff
@@ -668,6 +675,36 @@ func (l *LibvirtLXCBackend) Run(job *host.Job, runConfig *RunConfig) (err error)
 	return nil
 }
 
+// maybeResolveDiscoverdURI resolves a discoverd host in the given URI to an
+// address using the configured discoverd URL as the host is likely not using
+// discoverd to resolve DNS queries
+func (l *LibvirtLXCBackend) maybeResolveDiscoverdURI(uri string) (string, error) {
```

Comment:
`maybe` should only be used in a function name when it panics if there is an error.